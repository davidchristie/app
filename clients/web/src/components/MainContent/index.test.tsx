import { MainContent } from ".";
import { getByTestId, render, screen } from "../../testing";

describe("MainContent", () => {
  it("renders correctly", () => {
    const { container } = render(
      <MainContent>
        <div data-testid="children" />
      </MainContent>
    );
    const element = screen.getByTestId("MainContent");
    expect(getByTestId(element, "children")).toBeDefined();
    expect(element).toBe(container.firstChild);
    expect(element).toMatchSnapshot();
  });
});
