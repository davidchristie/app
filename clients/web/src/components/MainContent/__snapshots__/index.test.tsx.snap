// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent renders correctly 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthLg"
  data-testid="MainContent"
>
  <div
    class="MuiBox-root MuiBox-root-1"
  >
    <div
      data-testid="children"
    />
  </div>
</div>
`;
